<?xml version="1.0" encoding="UTF-8"?>
<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
    <caseValues>
        <caseType>Nominative</caseType>
        <plural>false</plural>
        <value>Fehler</value>
    </caseValues>
    <caseValues>
        <caseType>Nominative</caseType>
        <plural>true</plural>
        <value>Fehler</value>
    </caseValues>
    <caseValues>
        <caseType>Accusative</caseType>
        <plural>false</plural>
        <value>Fehler</value>
    </caseValues>
    <caseValues>
        <caseType>Accusative</caseType>
        <plural>true</plural>
        <value>Fehler</value>
    </caseValues>
    <caseValues>
        <caseType>Genitive</caseType>
        <plural>false</plural>
        <value>Fehlers</value>
    </caseValues>
    <caseValues>
        <caseType>Genitive</caseType>
        <plural>true</plural>
        <value>Fehler</value>
    </caseValues>
    <caseValues>
        <caseType>Dative</caseType>
        <plural>false</plural>
        <value>Fehler</value>
    </caseValues>
    <caseValues>
        <caseType>Dative</caseType>
        <plural>true</plural>
        <value>Fehlern</value>
    </caseValues>
    <fields>
        <help>Kontext, der den Fehler verursacht hat, sofern bekannt</help>
        <label>Kontexttyp</label>
        <name>Context_Type__c</name>
    </fields>
    <fields>
        <help>Datum und Uhrzeit bei Auftreten des Fehlers</help>
        <label>Datum/Uhrzeit</label>
        <name>Datetime__c</name>
    </fields>
    <fields>
        <help>Gibt an, ob zu diesem Fehler eine E-Mail-Benachrichtigung gesendet wurde</help>
        <label>E-Mail gesendet</label>
        <name>Email_Sent__c</name>
    </fields>
    <fields>
        <help>Typ des aufgetretenen Fehlers</help>
        <label>Fehlertyp</label>
        <name>Error_Type__c</name>
    </fields>
    <fields>
        <help>Vollständiger Text der Fehlermeldung</help>
        <label>Vollständige Nachricht</label>
        <name>Full_Message__c</name>
    </fields>
    <fields>
        <help>Objekttyp, zu dem der Fehler aufgetreten ist, sofern bekannt</help>
        <label>Objekttyp</label>
        <name>Object_Type__c</name>
    </fields>
    <fields>
        <help>Gibt an, ob der Fehler in Chatter gepostet wurde.</help>
        <label>Gepostet in Chatter</label>
        <name>Posted_in_Chatter__c</name>
    </fields>
    <fields>
        <help>Link zum Datensatz, der den Fehler verursacht hat, sofern verfügbar.</help>
        <label>Datensatz-URL</label>
        <name>Record_URL__c</name>
    </fields>
    <fields>
        <help>Die Salesforce-ID des Datensatzes, der diesem Fehler zugeordnet ist.</help>
        <label>Verwandte Datensatz-ID</label>
        <name>Related_Record_ID__c</name>
    </fields>
    <fields>
        <help>Diese Option zeigt an, dass der Datensatz in &quot;Verwandte Datensatz-ID&quot; bei der nächsten planmäßigen Verarbeitung erneut verarbeitet wird. Gelingt dies, wird das Feld deaktiviert. Gilt für bestimmte Fehlertypen, die in geplanten Prozessen auftreten.</help>
        <label>Ausstehende wiederholen?</label>
        <name>Retry_Pending__c</name>
    </fields>
    <fields>
        <help>Stapelverfolgung für den ausgegebenen Fehler, falls bei Laufzeit verfügbar</help>
        <label>Stapelverfolgung</label>
        <name>Stack_Trace__c</name>
    </fields>
    <gender>Masculine</gender>
    <layouts>
        <layout>Error Layout</layout>
        <sections>
            <label>Benutzerdefinierte Links</label>
            <section>Custom Links</section>
        </sections>
    </layouts>
    <nameFieldLabel>Fehlernummer</nameFieldLabel>
</CustomObjectTranslation>
